### PR TITLE
New package: disklow-1.2

### PR DESCRIPTION
**Pull request by Loomsday**
#### General
- [ x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/disklow/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+homepage="https://loomsday.co.nz/development?id=linuxutils"
+distfiles="https://loomsday.co.nz/sources/disklow-${version}.tar.gz"
+checksum=05b9f510278147f24b0556eb745ff2b71c98c00fedf33434744fbe76ec884c26
+
+do_install() {
+	vbin  disklow
+	vconf disklow.conf
+	vman  disklow.1
+	vdoc  sendmail-setup.txt
+	}
```
Comment (ericonr) [inline srcpkgs/disklow/template:18]: Fix indentation

File: srcpkgs/disklow/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'disklow'
+pkgname=disklow
+version=1.2
+revision=1
+depends="perl perl-Filesys-Df perl-Config-General perl-Mail-Sendmail perl-Net-SMTP-SSL perl-Authen-SASL"
```
Comment (ericonr) [inline srcpkgs/disklow/template:5]: Please break at ~80 columns

Comment (ericonr) [inline srcpkgs/disklow/template:5]: I think you don't need to depend on `perl` directly once you depend on all those.

**Review by ericonr — COMMENTED** [review]
My only issue would be that this is a single script that could also live in a user's home directory. What are your thoughts?

**Comment by Loomsday** [comment]
> My only issue would be that this is a single script that could also live in a user's home directory. What are your thoughts?

Thanks for your review.

Disklow is a script that monitors the disk space system-wide. Thus, it belongs to the system (/bin), not in a user directory.

File: srcpkgs/disklow/template (hunk 2 above)
Comment (Loomsday) [reply srcpkgs/disklow/template]: Removed "perl"

File: srcpkgs/disklow/template (hunk 1 above)
Comment (Loomsday) [reply srcpkgs/disklow/template]: Done.

**Pull request merged**